### ISSUE TITLE
Intellij Plugin returns "Invalid fields: verticalMultiline"

### ISSUE THREAD

**Issue by kantlove** [scalameta/scalafmt#1232]
- **Version**: 1.5.1
- **Integration**: IntelliJ IDEA Community 2018.1.5
- **Configuration**:
```
maxColumn = 100
docstrings = ScalaDoc
includeCurlyBraceInSelectChains = true
danglingParentheses = true

continuationIndent {
  callSite = 2
  defnSite = 2
}

verticalMultiline {
  atDefnSite = true
  newlineBeforeImplicitKW = false
  newlineAfterImplicitKW = true
  newlineAfterOpenParen = true
}
```

Without the `verticalMultiline` config, it works fine.

**Comment by olafurpg** [comment]
Thanks for reporting! This option has only been available since v1.6.0-RC1 so it's expected that it errors on 1.5.1. I've updated the website to clarify when `verticalMultiline` got added https://github.com/scalameta/scalafmt/commit/fc9bf636fb701c035c7765fb888efbbfe15d3204

**Comment by robbyki** [comment]
I'm using 1.6.0-RC4 both for my Intellij plugin and for my sbt plugin but I still get:
```
Failed to load scalafmt config .scalafmt.conf: Invalid fields: verticalMultiline Using default configuration instead.
```
